test/functional/mempool_accept.py

```diff
@@ -128,7 +111,7 @@ def run_test(self):
         tx.deserialize(BytesIO(hex_str_to_bytes(raw_tx_0)))
         tx.vout[0].nValue -= int(fee * COIN)  # Double the fee
         tx.vin[0].nSequence = BIP125_SEQUENCE_NUMBER + 1  # Now, opt out of RBF
-        raw_tx_0 = node.signrawtransactionwithwallet(tx.serialize().hex())['hex']
+        raw_tx_0 = miniwallet.send_self_transfer(from_node = node, fee_rate = fee * 2, send_tx = False, existing_tx = tx)['hex']
```

Comment:
Why is this call needed? The tx should already be signed

Reply:
I thought we changed the `nValue` and `nSequence` of raw_tx_0 therefore we have to resign it, is that not the case?

Reply:
miniwallet doesn't do any sighashing, so no. What is the error message you see when not "signing"?

Reply:
I commented 114 out and it fails this assertion
```
 self.check_mempool_result(
  result_expected=[{'txid': txid_0, 'allowed': True, 'vsize': tx.get_vsize(), 'fees': {'base': (2 * fee)}}],
  rawtxs=[raw_tx_0],
 )
```

Reply:
You can't comment out the line, but you can remove the "signing roundtrip" and simply take the tx as is.

Reply:
I'm sorry but do you mean just take the `raw_tx_0 = tx.serialize().hex()`?

Reply:
I tried the above approach but am unable to get/spend the utxo in line 149/151 associated with `txid_0`, because it doesnot have  tides with `send_self_transfer` . Is there any way around it?

Reply:
Maybe add a send_tx to miniwallet which does the needed steps?